middle.mkd

```diff
@@ -274,8 +274,9 @@ A GeoJSON object with the type "Feature" is a feature object.
 A GeoJSON object with the type "FeatureCollection" is a feature
 collection object. A feature collection object MUST have a member with
 the name "features". The value of "features" is an array. Each element
-of the array is a feature object as defined above. It is possible for
-this array to be empty.
+of the array MUST be a feature object as defined above. It is possible
+for this array to be empty. The values in the array are ordered, meaning
+that GeoJSON implementations MUST maintain order when processing them.
```

Comment:
hello tim.

On 2016-02-17 23:52, Tim Schaub wrote:
> <https://github.com/geojson/draft-geojson/pull/151#discussion_r53245187>:
>> +of the array MUST be a feature object as defined above. It is possible
>> +for this array to be empty. The values in the array are ordered, meaning
>> +that GeoJSON implementations MUST maintain order when processing them.
>     The values in the array are ordered
> Since a JSON array is "an /ordered sequence/ of zero or more values", I
> don't think it adds value to our spec to say that the values in the
> array are ordered.

yes, the JSON array is ordered. but that does not necessarily make the 
GeoJSON values serialized as an JSON array ordered. after all, the thing 
is called a "Collection", which typically are unordered. in this case 
it's ordered, as @adoyle found out, but that's helpful info and not 
self-evident.

>     GeoJSON implementations MUST maintain order when processing them
> I'm curious what is motivating this change.

it's making sure that devs don't think "it's a collection, so let's 
ignore order".